Lesson 9

nns0808 opened the pull request:
Created flexboxes for sections: experience, connection. Sticky navbar

nns0808 committed:
added branch 9

nns0808 committed:
add index

nns0808 committed:
Create css

nns0808 committed:
Delete css

nns0808 committed:
Create test

nns0808 committed:
add index.css

nns0808 committed:
Delete css/test

nns0808 committed:
Create test

nns0808 committed:
Add files via upload

nns0808 committed:
Delete images/test

nns0808 committed:
updated

nns0808 committed:
updated lesson-9

nns0808 committed:
updated lesson-9

nns0808 committed:
updated

nns0808 committed:
Merge branches 'lesson-9' and 'lesson-9' of github.com:nns0808/natalie-novikova-jupiter into lesson-9

jmhummer commented:
Natalie,

Sorry I'm so late in grading this week, the holidays have been a little busy for me. A few small things I noticed:

Your navigation bar's "About" link does not work, I think this is because the section's id is "About" instead of "about" (line 32).

The link for the "Connect" section works but I noticed that the id="connect" is placed on the <ul> tag (line 117) instead of the <section> tag (line 115) which is not consistent with your other sections. Changing will likely make your "connect" section look different. You can still have an id with the <ul> tag, just rename it to something different (perhaps "connect-links") and update your CSS to reflect that so that no changes occur.

If you add "justify-content: center" to your (now renamed) "#connect-links" section, your social media icons will center but I still see a bit of padding on the left side that shifts the icons to the right a little bit. If you specify the "padding-left: 0px" it should eliminate that padding without impacting your other sections.

When looking at your "Experience" section with a full screen, your exp-container turns into a 2x2 grid. To keep your experience as a single column of exp-items, you can change the width of exp-items to 100%. Your current setup can often be desirable if displaying a series of photos, users with larger screens see a grid of photos for easier navigation, but I think having a single column better fits the assignment requirements.

Let me know if you have any questions about this. If you made these changes feel free to merge and move on without my approval, I may not be able to do final approval until this weekend and I don't want you to have to wait.

Happy New Year!
Joe

nns0808 committed:
changed connect and experience sections, about

nns0808 committed:
changed lesson-9

nns0808 committed:
updated lesson-9

nns0808 committed:
updated lesson-9

nns0808 commented:
Hi Joe! I changed, merged and resubmitted my assignment Lesson-9.
Happy New Year!


On Wed, Jan 1, 2025 at 12:31 PM Joe Hummer ***@***.***> wrote:

> Natalie,
>
> Sorry I'm so late in grading this week, the holidays have been a little
> busy for me. A few small things I noticed:
>
> Your navigation bar's "About" link does not work, I think this is because
> the section's id is "About" instead of "about" (line 32).
>
> The link for the "Connect" section works but I noticed that the
> id="connect" is placed on the
>
>    tag (line 117) instead of the tag (line 115) which is not consistent
>    with your other sections. Changing will likely make your "connect" section
>    look different. You can still have an id with the
>       tag, just rename it to something different (perhaps
>       "connect-links") and update your CSS to reflect that so that no changes
>       occur.
>
>       If you add "justify-content: center" to your (now renamed)
>       "#connect-links" section, your social media icons will center but I still
>       see a bit of padding on the left side that shifts the icons to the right a
>       little bit. If you specify the "padding-left: 0px" it should eliminate that
>       padding without impacting your other sections.
>
>       When looking at your "Experience" section with a full screen, your
>       exp-container turns into a 2x2 grid. To keep your experience as a single
>       column of exp-items, you can change the width of exp-items to 100%. Your
>       current setup can often be desirable if displaying a series of photos,
>       users with larger screens see a grid of photos for easier navigation, but I
>       think having a single column better fits the assignment requirements.
>
>       Let me know if you have any questions about this. If you made these
>       changes feel free to merge and move on without my approval, I may not be
>       able to do final approval until this weekend and I don't want you to have
>       to wait.
>
>       Happy New Year!
>       Joe
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nns0808/natalie-novikova-jupiter/pull/4#issuecomment-2567137063>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BK2PMQL27YURKUNSUS2UALL2IRGBFAVCNFSM6AAAAABUFK2FOOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNRXGEZTOMBWGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Best regards,
Natalia.

nns0808 commented:
Hi Joe! I am wondering how these files were repeated. I deleted both
repeated html and css files.

Best,
Natalie

On Thu, Jan 2, 2025 at 9:09 AM Natalia Novikova ***@***.***> wrote:

> Hi Joe! I changed, merged and resubmitted my assignment Lesson-9.
> Happy New Year!
>
>
> On Wed, Jan 1, 2025 at 12:31 PM Joe Hummer ***@***.***>
> wrote:
>
>> Natalie,
>>
>> Sorry I'm so late in grading this week, the holidays have been a little
>> busy for me. A few small things I noticed:
>>
>> Your navigation bar's "About" link does not work, I think this is because
>> the section's id is "About" instead of "about" (line 32).
>>
>> The link for the "Connect" section works but I noticed that the
>> id="connect" is placed on the
>>
>>    tag (line 117) instead of the tag (line 115) which is not consistent
>>    with your other sections. Changing will likely make your "connect" section
>>    look different. You can still have an id with the
>>       tag, just rename it to something different (perhaps
>>       "connect-links") and update your CSS to reflect that so that no changes
>>       occur.
>>
>>       If you add "justify-content: center" to your (now renamed)
>>       "#connect-links" section, your social media icons will center but I still
>>       see a bit of padding on the left side that shifts the icons to the right a
>>       little bit. If you specify the "padding-left: 0px" it should eliminate that
>>       padding without impacting your other sections.
>>
>>       When looking at your "Experience" section with a full screen, your
>>       exp-container turns into a 2x2 grid. To keep your experience as a single
>>       column of exp-items, you can change the width of exp-items to 100%. Your
>>       current setup can often be desirable if displaying a series of photos,
>>       users with larger screens see a grid of photos for easier navigation, but I
>>       think having a single column better fits the assignment requirements.
>>
>>       Let me know if you have any questions about this. If you made
>>       these changes feel free to merge and move on without my approval, I may not
>>       be able to do final approval until this weekend and I don't want you to
>>       have to wait.
>>
>>       Happy New Year!
>>       Joe
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/nns0808/natalie-novikova-jupiter/pull/4#issuecomment-2567137063>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BK2PMQL27YURKUNSUS2UALL2IRGBFAVCNFSM6AAAAABUFK2FOOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNRXGEZTOMBWGM>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
> Best regards,
> Natalia.
>
>

-- 
Best regards,
Natalia.
